README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
```

Comment:
The problem here is that too many Tetris pieces are the same color. In your review, you might want to suggest that I choose colors that would be easier to see when playing the game.